Source/WebCore/page/Page.cpp

```diff
@@ -4224,6 +4225,15 @@ bool Page::shouldDisableCorsForRequestTo(const URL& url) const
     });
 }
 
+URL Page::urlForSelectedText()
```

Comment:
Can this function be `const`? I think this function should also be called `fragmentDirectiveURLForSelectedText()`